app/cdap/components/RecipeSteps/RecipeStepsTableComponent/index.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+import React from 'react';
+import { useStyles } from '../styles';
+import T from 'i18n-react';
+import { IRecipeStepTebleProps } from './types';
+import { DeleteIcon } from '../iconStore';
+
+export default function({ recipeSteps }: IRecipeStepTebleProps) {
+  const classes = useStyles();
+
+  return (
+    <TableContainer component={Box}>
+      <Table aria-label="recipe steps table">
+        <TableHead>
+          <TableRow className={classes.recipeStepsTableRowStyles}>
+            <TableCell
+              data-testid="serial-no"
+              classes={{ head: classes.recipeStepsTableHeadStyles }}
+            >
+              {T.translate('features.WranglerNewRecipeSteps.labels.serialNo')}
+            </TableCell>
+            <TableCell
+              data-testid="recipe-steps"
+              classes={{ head: classes.recipeStepsTableHeadStyles }}
+            >
+              {T.translate('features.WranglerNewRecipeSteps.labels.recipeSteps')}
```

Comment:
reduce 2 TableCell elements in to 1 TableCell element in TableHead

Reply:
done